tests/test_utc_tai_tdb_tt.py

```
@@ -39,430 +39,426 @@
 from julian.leap_seconds import set_ut_model
 
 
-class Test_utc_tai_tdb_tt(unittest.TestCase):
+def test_utc_tai_tdb_tt():
```

Ruff (0.15.6)

[warning] Too many branches (13 > 12)

(PLR0912)